src/bindings/c/ov/include/c_api/ov_c_api.h

```diff
@@ -0,0 +1,1122 @@
+    } status;
+
+    /**
+     * @brief The absolute time, in microseconds, that the node ran (in total).
+     */
+    int64_t real_time;
+
+    /**
+     * @brief The net host CPU time that the node ran.
+     */
+    int64_t cpu_time;
+
+    /**
+     * @brief Name of a node.
+     */
+    char* node_name;
+
+    /**
+     * @brief Execution type of a unit.
+     */
+    char* exec_type;
+
+    /**
+     * @brief Node type.
+     */
+    char* node_type;
+}ov_profiling_info_t;
+
+/**
+ * @struct ov_profiling_info_list_t
+ */
+typedef struct {
+    ov_profiling_info_t* profiling_infos;
+    size_t num;
+}ov_profiling_info_list_t;
+
+/**
+ * @enum ov_status_code_e
+ * @brief This enum contains codes for all possible return values of the interface functions
+ */
+typedef enum {
+    OK = 0,
+    GENERAL_ERROR = 1,
+    NOT_IMPLEMENTED = 2,
+    NETWORK_NOT_LOADED = 3,
+    PARAMETER_MISMATCH = 4,
+    NOT_FOUND = 5,
+    OUT_OF_BOUNDS = 6,
+    /*
+     * @brief exception not of std::exception derived type was thrown
+     */
+    UNEXPECTED = 7,
+    REQUEST_BUSY = 8,
+    RESULT_NOT_READY = 9,
+    NOT_ALLOCATED = 10,
+    INFER_NOT_STARTED = 11,
+    NETWORK_NOT_READ = 12,
+    INFER_CANCELLED = 13,
+    UNKNOWN_ERROR = 14,
+} ov_status_e;
+
+/**
+ * @struct ov_version_t
+ */
+typedef struct ov_version {
+    char* buildNumber;
+    char* description;
+}ov_version_t;
+
+typedef struct {
+    char* device_name;
+    char* buildNumber;
+    char* description;
+}ov_core_version_t;
+
+/**
+ * @struct ov_core_version_list_t
+ */
+typedef struct {
+    ov_core_version_t *versions;
+    size_t num_vers;
+}ov_core_version_list_t;
+
+/**
+ * @struct ov_core_t
+ */
+typedef struct ov_core ov_core_t;
+
+/**
+ * @struct ov_node_t
+ */
+typedef struct ov_node ov_node_t;
+
+/**
+ * @struct ov_output_node_t
+ */
+typedef struct ov_output_node ov_output_node_t;
+
+/**
+ * @struct ov_output_node_list_t
+ */
+typedef struct{
+    ov_output_node_t *output_nodes;
+    size_t num;
+} ov_output_node_list_t;
+
+/**
+ * @struct ov_model_t
+ */
+typedef struct ov_model ov_model_t;
+
+/**
+ * @struct ov_preprocess_t
+ */
+typedef struct ov_preprocess ov_preprocess_t;
+
+/**
+ * @struct ov_preprocess_input_info_t
+ */
+typedef struct ov_preprocess_input_info ov_preprocess_input_info_t;
+
+/**
+ * @struct ov_preprocess_input_tensor_info_t
+ */
+typedef struct ov_preprocess_input_tensor_info ov_preprocess_input_tensor_info_t;
+
+/**
+ * @struct ov_preprocess_output_info_t
+ */
+typedef struct ov_preprocess_output_info ov_preprocess_output_info_t;
+
+/**
+ * @struct ov_preprocess_output_tensor_info_t
+ */
+typedef struct ov_preprocess_output_tensor_info ov_preprocess_output_tensor_info_t;
+
+/**
+ * @struct ov_preprocess_input_model_info_t
+ */
+typedef struct ov_preprocess_input_model_info ov_preprocess_input_model_info_t;
+
+/**
+ * @struct ov_preprocess_input_process_steps_t
+ */
+typedef struct ov_preprocess_input_process_steps ov_preprocess_input_process_steps_t;
```

Comment:
don't we need `ov_preprocess_output_model_info` and `ov_preprocess_output_process_steps `?